client/connector.go

```diff
@@ -198,6 +199,56 @@ func (c *connector) Insert(tableName string, columnNames []string, rows []Row, u
 	return numRows, nil
 }
 
+// hllHash computes hash value from original value
+func hllHash(dataType memCom.DataType, value interface{}) (uint64, error) {
+	var out [2]uint64
+	var ok bool
+	numBytes := 4
+	switch dataType {
+	case memCom.UUID:
+		out, ok = memCom.ConvertToUUID(value)
+		if ok {
+			return out[0] ^ out[1], nil
+		}
+	case memCom.Uint32:
+		value, ok = memCom.ConvertToUint32(value)
+		numBytes = 4
+	case memCom.Int32:
+		value, ok = memCom.ConvertToInt32(value)
+		numBytes = 4
+	case memCom.Int64:
+		value, ok = memCom.ConvertToInt64(value)
+		numBytes = 8
+	default:
+		return 0, utils.StackError(nil, "Unsupported data type for hyperloglog computation")
+	}
+	if !ok {
+		return 0, utils.StackError(nil, "Invalid %v for data type %s", memCom.DataTypeName[dataType])
+	}
+	out = utils.Murmur3Sum128(unsafe.Pointer(&value), numBytes,  0)
```

Comment:
do we only use first 64 bit? then just return uint64?

Reply:
Done